pandas/io/formats/style.py

```diff
@@ -1006,19 +1006,23 @@ def hide_columns(self, subset) -> "Styler":
     def _highlight_null(v, null_color: str) -> str:
         return f"background-color: {null_color}" if pd.isna(v) else ""
 
-    def highlight_null(self, null_color: str = "red") -> "Styler":
+    def highlight_null(self, null_color: str = "red", subset=None) -> "Styler":
         """
         Shade the background ``null_color`` for missing values.
 
         Parameters
         ----------
         null_color : str
```

Comment:
Should the default (red) be mentioned here in the docstring?

Reply:
sure~